File: Ubistart-FullStack-Challenge.Domain/Dtos/TaskDto.cs
using System;

namespace Ubistart_FullStack_Challenge.Domain.Dtos
{
	public class TaskDto
	{
		public int IdTask { get; set; }
		public DateTime InsertionDate { get; set; }
		public string Description { get; set; }
		public DateTime Deadline { get; set; }
		public DateTime FinishDate { get; set; }
		public DateTime EditDate { get; set; }
		public int UserFK { get; set; }
		public string Email { get; set; }
		public TaskDto(DateTime insertionDate, string description, DateTime deadline)
		{
			InsertionDate = insertionDate;
			Description = description;
			Deadline = deadline;
		}
		public TaskDto() { }
	}
}

File: Ubistart-FullStack-Challenge/ClientApp/src/app/dtos/TaskDto.ts
import * as internal from "assert";
import { TaskDisplayDto } from "./TaskDisplayDto";

export class TaskDto{
    public idTask: string;
    public insertionDate: Date;
    public description: string;
    public deadline: Date;
    public finishDate: Date;
    public editDate: Date;
    public userFk: string;
    public email: string;

    constructor(displayTask: TaskDisplayDto){
        this.idTask = displayTask.id;
        this.insertionDate = displayTask.insertionDate;
        this.description = displayTask.description;
        this.deadline = displayTask.deadline;
        this.finishDate = displayTask.finishDate;
        this.editDate = displayTask.editDate;
    }
}